coderabbit.yaml

```
@@ -0,0 +1,60 @@
+        - Component reusability and organization
+        - TypeScript usage and type safety
+        - Import path cleanliness
+        - UI/UX considerations
+
+    - path: "backend/**"
+      instructions: |
+        For backend changes, also focus on:
+        - Django app configuration
+        - API design and security
+        - Database model relationships
+        - Error handling and validation
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)